File: apps/admin/lib/v2-design-system.ts
/**
 * CarePoint V2 shared design foundations.
 * Phase 0 declaration only: intentionally not wired into existing screens.
 */
export const carePointV2Colors = {
  surface: "#FAF8FF",
  canvas: "#F8FAFC",
  card: "#FFFFFF",
  surfaceLow: "#F2F3FF",
  surfaceHigh: "#E2E7FF",
  ink: "#131B2E",
  muted: "#6E7881",
  outline: "#D8E0EA",
  clinicalBlue: "#0EA5E9",
  clinicalBlueDeep: "#006591",
  neuralViolet: "#8B5CF6",
  cyanSignal: "#22D3EE",
  midnight: "#0F172A",
  careGreen: "#10B981",
  attention: "#F59E0B",
  safety: "#FB7185",
} as const;

export const carePointV2Spacing = { x1: 8, x2: 16, x3: 24, x4: 32, x5: 40, x6: 48 } as const;
export const carePointV2Radius = { control: 12, card: 22, overlay: 28 } as const;
export const carePointV2Modalities = ["CLINIC", "TELEMEDICINE", "HOME_VISIT", "TRANSPORT"] as const;
export type CarePointV2Modality = (typeof carePointV2Modalities)[number];
export const carePointV2Accessibility = {
  statusRequiresText: true,
  statusRequiresNonColorSignal: true,
  criticalDataUsesSolidSurface: true,
  rtlSupported: true,
} as const;

File: packages/mobile_core/lib/v2_design_system.dart
import 'package:flutter/material.dart';

/// CarePoint V2 shared design foundations.
/// Phase 0 declaration only; existing user-facing screens do not import it.
abstract final class CarePointV2DesignTokens {
  static const Color surface = Color(0xFFFAF8FF);
  static const Color canvas = Color(0xFFF8FAFC);
  static const Color card = Color(0xFFFFFFFF);
  static const Color surfaceLow = Color(0xFFF2F3FF);
  static const Color surfaceHigh = Color(0xFFE2E7FF);
  static const Color ink = Color(0xFF131B2E);
  static const Color muted = Color(0xFF6E7881);
  static const Color outline = Color(0xFFD8E0EA);
  static const Color clinicalBlue = Color(0xFF0EA5E9);
  static const Color clinicalBlueDeep = Color(0xFF006591);
  static const Color neuralViolet = Color(0xFF8B5CF6);
  static const Color cyanSignal = Color(0xFF22D3EE);
  static const Color midnight = Color(0xFF0F172A);
  static const Color careGreen = Color(0xFF10B981);
  static const Color attention = Color(0xFFF59E0B);
  static const Color safety = Color(0xFFFB7185);

  static const double spacing1 = 8;
  static const double spacing2 = 16;
  static const double spacing3 = 24;
  static const double spacing4 = 32;
  static const double spacing5 = 40;
  static const double spacing6 = 48;
  static const double controlRadius = 12;
  static const double cardRadius = 22;
  static const double overlayRadius = 28;
}

abstract final class CarePointV2Modality {
  static const String clinic = 'CLINIC';
  static const String telemedicine = 'TELEMEDICINE';
  static const String homeVisit = 'HOME_VISIT';
  static const String transport = 'TRANSPORT';
  static const Set<String> values = {clinic, telemedicine, homeVisit, transport};
}

abstract final class CarePointV2Accessibility {
  static const bool statusRequiresText = true;
  static const bool statusRequiresNonColorSignal = true;
  static const bool criticalDataUsesSolidSurface = true;
  static const bool rtlSupported = true;
}
